Makefile

```diff
@@ -2,6 +2,7 @@ PYODIDE_ROOT=$(abspath .)
 include Makefile.envs
 
 FILEPACKAGER=$(PYODIDE_ROOT)/tools/file_packager.py
+ABI=1997
```

Comment:
Let's change this to `PYODIDE_PACKAGE_ABI` so it's more clear what we're talking about.